app/controllers/account/email_passwords_controller.rb

```diff
@@ -45,7 +45,7 @@ def resend_email_change
   def cancel_email_change
     current_user.cancel_email_change
 
-    redirect_back_or_to root_path, notice: "You have cancelled your pending email address change."
+    redirect_to account_path, notice: "You have cancelled your pending email address change."
```

Comment:
Is there a reason you've changed from `redirect_back_or_to` to `redirect_to`?

Reply:
I might have understood `redirect_back_or_to` wrong. It only seemed to redirect back to the "Change your email or password" page, anyway. So this change was more accurately stopping `cancel_email_change` redirecting back to either the Dashboard or the Change your email or password page, and redirecting unconditionally to Account instead.

Not necessarily the right commit to make that change in. But if what I'm saying makes sense, at the very least I can explain it in the commit message.

Reply:
Ah, OK, that makes sense to me now. Thanks for explaining :+1: